### PR TITLE
Add a timeout knob to control stackdriver exporter behavior.

### PR DESCRIPTION
**Pull request by yegle**
The default timeout is `5s`.

Works towards google/exposure-notifications-verification-server#474

### CODE REVIEW

File: pkg/observability/config.go [hunk 1]
```diff
@@ -74,4 +74,5 @@ type StackdriverConfig struct {
 	ReportingInterval    time.Duration `env:"STACKDRIVER_REPORTING_INTERVAL, default=2m"`
 	BundleDelayThreshold time.Duration `env:"STACKDRIVER_BUNDLE_DELAY_THRESHOLD, default=2s"`
 	BundleCountThreshold uint          `env:"STACKDRIVER_BUNDLE_COUNT_THRESHOLD, default=50"`
+	Timeout              time.Duration `env:"STACKDRIVER_TIMEOUT, default=5s"`
```
Comment (icco) [inline pkg/observability/config.go:77]: Are you using this anywhere?

Comment (yegle) [reply pkg/observability/config.go:77]: Ah forgot about that. Fixed.

**Review by mikehelmick — APPROVED** [review]

**Review by icco — APPROVED** [review]
/lgtm

**Pull request merged**